sass/cd/cd-header/_cd-user-menu.scss

```diff
@@ -3,12 +3,6 @@
 //
 // Includes user-centered settings and/or actions.
 //
-// Notes:
-// - menu items can be links or buttons with dropdowns
-// - dropdown menu items can be given the class .cd-user-menu__item--small to
-//   restrict their width and centre the dropdown
-// - add class .cd-user-menu__item--mobile-icon to menu items to only show the
-//   icon on mobile.
 //——————————————————————————————————————————————————————————————————————————————
```

Comment:
@rupl FYI I removed these comments and the facility for the icon only on mobile. It won't be easy to do this since we are not longer using a separate html element, but instead a pseudo element. I suggest we wait until it becomes relevant